### ISSUE TITLE
Extra space in username

### ISSUE THREAD

**Issue by fearoffish** [blinksh/blink#211]
When I set up a host in the config with a username, when I try and connect to it there is an extra space (and yes I've checked if there actually is one in the username!)

![screen shot 2016-12-17 at 18 30 00](https://cloud.githubusercontent.com/assets/79/21288874/e097f7ac-c486-11e6-8db0-bdefe68ae644.png)

**Comment by cgull** [comment]
I've just encountered this myself on an iPad 3 iOS 9.3.1. For me it was caused by setting my username `cgull` as default in Config. Since that's not an English word, iOS will autocorrect to `chill` unless I accept the verbatim quoted completion. When I do that, the text field does not show a trailing space, but if i select it and move the cursor to the end I can backspace and a non visible character is deleted.  If I don't delete it I get the same issue shown in the screenshot.

Perhaps the text field is getting trimmed or right-shifted for display, hiding the space?
